microlith/prometheus/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -91,3 +91,39 @@ groups:
         title: "Metadata Overhead Above 90% on Bucket: {{ $labels.bucket }}, Node: {{ $labels.instance }}"
         description: The percentage of memory that is taken up by metadata is over 90%
         remediation: Increase memory allocation for bucket or change the evictionPolicy of the bucket from `Value-only` (be aware this will have an adverse effect on performance).
+
+    - alert: CB90061-diskWriteQueueLength-Alert
+      expr: |
+        kv_ep_storage_age_seconds > 60
+      for: 1m
+      labels:
+        job: couchbase_prometheus
+        kind: bucket
+        health_check_id: CB90061
+        health_check_name: diskWriteQueueLength
+        cluster: '{{ $labels.cluster }}'
+        node: '{{ $labels.instance }}'
+        bucket: '{{ $labels.bucket }}'
+        severity: critical
+      annotations:
+        title: "The age of items in the disk write queue is very high on Bucket: {{ $labels.bucket }}, Node: {{ $labels.instance }}"
+        description: The age of items in the disk write queue is over 60 seconds.
+        remediation: Contact Couchbase Technical Support.
+
+    - alert: CB90061-diskWriteQueueLength-Warn
+      expr: |
+        kv_ep_storage_age_seconds > 40 < 60
```

Comment:
Have you verified that this syntax works?

Reply:
I have not as I was suggested it earlier. Would there need to be a boolean and operation between the > and < for that to work?

Reply:
I plugged it into Grafana and it does appear to work: 
![image](https://user-images.githubusercontent.com/2904440/151190112-38a61057-b4a0-4a0b-87b0-cf5107a52792.png)

(Having said that, it's worth checking your PromQL before committing it, even if it's based on someone else's suggestion, as there's currently a gap in our test coverage there so mistakes won't get caught.)